.gitignore

```diff
@@ -37,3 +37,15 @@ _sketches.cpp
 
 # local override files
 Makefile.local
+
+# downloaded package sources
+pkg/cmsis-dsp/cmsis-dsp
+pkg/libcoap/libcoap
+pkg/libfixmath/libfixmath
```

Comment:
I don't get these directories when building tests/libfixmath - actually this line seems to be completely superfluous.